lib/sites.js

```diff
@@ -27,10 +27,16 @@ Sites.prototype.getMeta = function( site_id, meta_key ) {
 			.get( '/sites/' + site_id + '/meta/' + meta_key )
 			.end( function( err, res ) {
 				if ( err ) {
+					// Meta doesn't exist, but that shouldn't trigger an error
+					if ( 404 === err.status ) {
+						return resolve( null );
+					}
+
 					return reject( err );
 				}
 
-				return resolve( res.body );
+				// Meta keys are unique per site
+				return resolve( res.body.data[0].meta_value );
```

Comment:
Ah interesting - I hadn't actually considered we could just return the value 😛 

I think this is the best way...we _could_ return the full meta object, but that's kinda lame for the calling code too. The only thing we really miss out on here is the meta id, but honestly that is a behind-the-scenes detail that we shouldn't care about, and don't need to b/c all meta operations can be done simply by key.

This will be much nicer.

Reply:
Much nicer: https://github.com/Automattic/vip-go-crontab/pull/13/commits/dc323c65896c9528b336e8e61206cf7f3805b4b6.